app/src/main/java/com/github/se/eduverse/ui/dashboard/DashboardScreen.kt

```diff
@@ -65,11 +72,18 @@ fun DashboardScreen(
   Scaffold(
       topBar = {
         TopAppBar(
+            modifier =
+                Modifier.fillMaxWidth()
+                    .background(
+                        Brush.horizontalGradient(
+                            colors = listOf(md_theme_light_secondary, md_theme_light_primary))),
```

Comment:
Here you should use `MaterialTheme.colorScheme.primary/secondary` to avoid inconsistence in dark mode

Reply:
As that part of the code was subject to conflicts coming from #271 I changed it while resolving them